Makefile

```diff
@@ -28,6 +28,14 @@ check-html:
 format-html:
 	npx prettier --plugin=prettier-plugin-jinja-template --parser=jinja-template --tab-width=2 --html-whitespace-sensitivity css --bracket-same-line=true --print-width=240 --write **/*.html
 
+.PHONY: format-css
+format-css:
+	npx prettier --tab-width=4 --write **/*.scss
+
+.PHONY: format-frontend
+format-frontend: format-css format-html
```

Comment:
Should we be formatting js files as well? We only have one but it's currently stored with 2 spaces, so immediately feels a little out of sync. Would be good to know that js will just be taken care of if/when we start writing more.